Makefile

```diff
@@ -13,11 +13,23 @@ endif
 BUILD_DATE ?= $(shell date -u '+%Y-%m-%d-%H:%M UTC')
 VERSION_FLAGS := -ldflags='-X "main.buildVersion=$(BUILD_VERSION)" -X "main.buildTime=$(BUILD_DATE)"'
 
+VERSION ?= 1.1.9
+
 # Deployment options
-K8SFS_PATH ?= ${HOME}/.k8sfs
-KUBE_PATH ?= ${K8SFS_PATH}/kubernetes
+HPK_MASTER_PATH ?= ${HOME}/.hpk-master
+KUBE_PATH ?= ${HPK_MASTER_PATH}/kubernetes
 EXTERNAL_DNS ?= 8.8.8.8
 
+
+K3S_REGISTRY_NAME ?= giannispetsis
+K3S_IMAGE_TAG=$(REGISTRY_NAME)/hpk-master:$(VERSION)
+
+PAUSE_REGISTRY_NAME ?= malvag
+PAUSE_IMAGE_TAG=$(PAUSE_REGISTRY_NAME)/pause:$(VERSION)
+
```

Comment:
* Use one `REGISTRY_NAME` variable, set it to `carvicsforth` as default.
* Remove the `print` target.